lib/rules/no-dupe-class-members.js

```diff
@@ -34,7 +34,7 @@ module.exports = {
          */
         function getState(name, isStatic) {
             const stateMap = stack[stack.length - 1];
-            const key = "$" + name; // to avoid "__proto__".
+            const key = `$${name}`; // to avoid "__proto__".
```

Comment:
Yes, look at the removed line- it has a $ prefix. (That said, if I need to
escape it and I'm doing it wrong, please let me know!)

On Sep 1, 2016 11:36 PM, "Kai Cataldo" <notifications@github.com> wrote:

> In lib/rules/no-dupe-class-members.js
> <https://github.com/eslint/eslint/pull/6989#discussion_r77292146>:
>
> > @@ -34,7 +34,7 @@ module.exports = {
> >           */
> >          function getState(name, isStatic) {
> >              const stateMap = stack[stack.length - 1];
> > -            const key = "$" + name; // to avoid "__proto__".
> > +            const key = `$${name}`; // to avoid "__proto__".
>
> Looks like there are two $ here - is that intentional?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/eslint/eslint/pull/6989/files/d0f454703eb527a515a55bbe2d2680ebc72d5382#r77292146>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AARWer5RnEHKtqsU5YsYevq_aQAVn5rVks5ql6eygaJpZM4JulxS>
> .
>